CoreMetrics/CoreMetrics.cs

```diff
@@ -60,13 +60,7 @@ private JObject GetBlocksTime(uint nBlocks, uint lastHeight)
                 JObject json = new JObject();
                 return json["error"] = "Requested number of blocks timestamps " + nBlocks + " exceeds quantity of known blocks " + Blockchain.Singleton.Height;
             }
-
-            if (nBlocks <= 0)
```

Comment:
==?

Reply:
@shargon  Uint can't be < 0,  because input -1 means ```nBlocks``` = uint 4294967295.

Reply:
![image](https://user-images.githubusercontent.com/38493437/68577180-8aed9980-04aa-11ea-8117-8225682b0daf.png)

Reply:
> UInt can't be < 0, because input -1 means nBlocks = uint 4294967295.

That's the reason for change to == 0.

Reply:
> > UInt can't be < 0, because input -1 means nBlocks = uint 4294967295.
> 
> That's the reason for change to == 0.

Oh, got it!